### ISSUE TITLE
Regression on 2.6.0 when passing base64 string

### ISSUE THREAD

**Issue by arashm** [lebedev-yury/carrierwave-base64#65]
I have this factory in my tests that works fine in 2.5.3 with no error:

```ruby
FactoryGirl.define do
  factory :screenshot do
    file "data:image/jpeg;base64,#{Base64.encode64(File.read(Rails.root.join('spec/fixtures/images/screenshot.jpg')))}"
  end
end
```
With version 2.6.0 they're all failing with:

```
     ArgumentError:                                                                                                                                                                            
       invalid base64                                                                                                                                                                          
```

What am I missing?

**Comment by sjdrabbani** [comment]
Use ``Base64.strict_encode64`` instead of ``Base64.encode64``.

**Comment by arashm** [comment]
If the gem used to work with `RFC 2045` and now it is not, I think it is still a regression. What's the reason behind this change? Also if this is the case, `2.6.0` should be a major `3.0.0` actually.

**Comment by lebedev-yury** [comment]
@arashm I think you are right. First - that was really a breaking change, which means `3.0.0`. Second - I really don't see a reason to switch to `RFC 4648` standard for base64 strings.

So I would rather remove this change, and release `2.6.1` as a fix. What do you think?

**Comment by arashm** [comment]
Sounds like a plan. As long as the public API is untouched.

**Comment by lebedev-yury** [comment]
ok, this issue is fixed in 2.6.1

@arashm thank you for reporting!
